### PR TITLE
[excel, word] (Preview APIs) Fixing the Preview API tables for Excel and Word

### PR DESCRIPTION
**Pull request by AlexJerabek**
The table was copied in wrong from the What's New tool. This PR addresses that.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[761ec78](https://github.com/OfficeDev/office-js-docs-pr/pull/1072/commits/761ec784d2c5300fecf01f04177ba15942d8b610)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/requirement-sets/word-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WordTable/docs/reference/requirement-sets/word-api-requirement-sets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C11%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906111746364138-1072%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=XB1JRUiAvDacwi4bK%2FjCENnxIC9WsYjccqO9wvaaxZQ%3D&st=2019-06-11T17%3A42%3A39Z&se=2019-07-12T17%3A47%3A39Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by AlexJerabek** [comment]
Fixed the table generation tool to exclude LoadOptions-related classes, as well as string-literal overload methods.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[edc2287](https://github.com/OfficeDev/office-js-docs-pr/pull/1072/commits/edc2287ab2d773115bf28d0cf1f2dfad7b72a1af)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/requirement-sets/excel-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WordTable/docs/reference/requirement-sets/excel-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/word-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WordTable/docs/reference/requirement-sets/word-api-requirement-sets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C14%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906141945309304-1072%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=1Hodvz4XO1H%2FqTXlR8kIdVXBvzX7b1%2BdKQ45dfYfpL0%3D&st=2019-06-14T19%3A41%3A38Z&se=2019-07-15T19%3A46%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/requirement-sets/excel-api-requirement-sets.md [hunk 1]
```diff
@@ -172,7 +170,7 @@ The following is a complete list of APIs currently in preview.
 ||[setDefault(newDefaultStyle: SlicerStyle \| string)](/javascript/api/excel/excel.slicerstylecollection#setdefault-newdefaultstyle-)|Sets the default SlicerStyle for use in the parent object's scope.|
 |[Table](/javascript/api/excel/excel.table)|[clearStyle()](/javascript/api/excel/excel.table#clearstyle--)|Changes the table to use the default table style.|
 ||[onFiltered](/javascript/api/excel/excel.table#onfiltered)|Occurs when filter is applied on a specific table.|
-|[TableCollection](/javascript/api/excel/excel.tablecollection)|[onFiltered](/javascript/api/excel/excel.tablecollection#onfiltered)|Occurs when filter is applied on any table in a workbook or worksheet.|
+|[TableCollection](/javascript/api/excel/excel.tablecollection)|[onFiltered](/javascript/api/excel/excel.tablecollection#onfiltered)|Occurs when filter is applied on any table in a workbook, or a worksheet.|
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/requirement-sets/excel-api-requirement-sets.md]: Wasn't the before text correct?

Comment (AlexJerabek) [reply docs/reference/requirement-sets/excel-api-requirement-sets.md]: It was, which means it got changed upstream. I'll fix it here and figure out what to do for DefinitelyTyped.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[d934e57](https://github.com/OfficeDev/office-js-docs-pr/pull/1072/commits/d934e578c19886ed22d4aa3997a2056f8e8aae44)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/requirement-sets/excel-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WordTable/docs/reference/requirement-sets/excel-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/word-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WordTable/docs/reference/requirement-sets/word-api-requirement-sets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C14%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906142048521640-1072%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=cPVQpQtLeFj4w6nxQBgA41GYEWVvb%2BVLInWLm6ahfUY%3D&st=2019-06-14T20%3A45%3A01Z&se=2019-07-15T20%3A50%3A01Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**